flexmeasures/api/v3_0/health.py

```diff
@@ -14,6 +15,19 @@ def _check_sql_database():
         return False
 
 
+def _check_redis() -> bool:
+    """Check status of the redis instance
+
+    :return: True if the redis instance is active, False otherwise
+    :rtype: bool
```

Comment:
I realize this is part of the formal sphinx docstring syntax, but I never understood the added value over the type annotation in the function signature, and only perceive it as a risk of mismatched documentation. Is there any advantage in including these return types, or has it just become archaic since pep 484?

For example, [this package](https://pypi.org/project/sphinx-autodoc-typehints/) suggests to me that the rtype (and also specifying param types in docstrings) has become obsolete since type annotations were introduced. I like how the second code blocks supplants the first code block.

Reply:
Indeed! I like not to use it as well, the simpler the better :smile:. I've just removed it.